skein/model.py

```diff
@@ -1038,23 +1038,26 @@ class Container(Base):
         The current container state.
     yarn_container_id : str
         The YARN container id.
+    yarn_node_address : str
+        The YARN node address given as ``host:port``.
     start_time : datetime
         The start time, None if container has not started.
     finish_time : datetime
         The finish time, None if container has not finished.
     """
     __slots__ = ('service_name', 'instance', '_state', 'yarn_container_id',
-                 'start_time', 'finish_time')
+                 'yarn_node_address', 'start_time', 'finish_time')
     _params = ('service_name', 'instance', 'state', 'yarn_container_id',
                'start_time', 'finish_time')
```

Comment:
Need to also add it to `_params`

Reply:
How about removing `_params` here? `_params == __slots__` so `_get_params` would behave as expected.

Reply:
> `_params == __slots__` so `_get_params`

There's a difference between `'_state'` vs `state'`.